code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -352,9 +359,26 @@
 	R.my_atom = src.emag
 	R.add_reagent("beer2", 50)
 	src.emag.name = "Mickey Finn's Special Brew"
-	return
-
 
+/obj/item/weapon/robot_module/butler/recharge_consumable(var/mob/living/silicon/robot/R)
```

Comment:
Turns out i need to implement respawn_con anyway since the wrap disappears which i didn't notice and restock doesn't currently apply to package wrap, just coil and medical stacks as shown here https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/game/objects/items/stacks/stack.dm#L342
I'll fix it momentarily, just finding out whats redundant and whats not.